src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj

```diff
@@ -42,6 +42,39 @@
     <RootNamespace>MongoDB.Driver</RootNamespace>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <NoWarn>NU5100</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="snappy64.dll" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <SnappyBinaries>Core/Compression/Snappy/lib/win</SnappyBinaries>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="$(SnappyBinaries)/snappy64.dll">
+      <Pack>true</Pack>
+      <PackagePath>x64/native/windows</PackagePath>
+    </Content>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="$(SnappyBinaries)/snappy64.dll">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Link>snappy64.dll</Link>
+    </Content>
+  </ItemGroup>
+  
+  <ItemGroup>
+    <Content Include="MongoDB.Driver.Core.targets">
+      <Pack>true</Pack>
+      <PackagePath>build</PackagePath>
+    </Content>
+  </ItemGroup>
+
```

Comment:
As I mentioned in our meeting I am deferring to Vincent and Dmitry on these csproj file changes. I don't understand them.